woocommerce/woocommerce-ios
# [Orders] Remove Order Creation feedback banner

Closes: #11433


## Description
As a follow-up from p91TBi-aPz-p2, this PR removes the Order Creation feedback banner:

<img width="365" alt="Screenshot 2023-12-29 at 09 21 04" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/f3f77792-2254-4904-b8f4-7e6baa849f46">

## Testing instructions
- Delete and re-install the app.
- Go to Orders, observe that no "Order Creation" banner shows up.
- Turn off the network, re-run the app.
- Go to Orders, observe that the network connectivity banner still appears
